Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm

```diff
@@ -3609,8 +3609,18 @@ - (id)accessibilityAttributeValue:(NSString*)attribute forParameter:(id)paramete
         });
     }
 
-    if ([attribute isEqualToString:AXAttributedStringForTextMarkerRangeAttribute])
+    if ([attribute isEqualToString:AXAttributedStringForTextMarkerRangeAttribute]) {
+#if ENABLE(AX_THREAD_TEXT_APIS)
+        // FIXME: Expand this beyond static text (i.e. ranges that span multiple objects).
+        if (AXObjectCache::useAXThreadTextApis() && backingObject->isStaticText()) {
+            if (!textMarkerRange)
```

Comment:
this could be moved outside the ENABLE(AX_THREAD) block

Reply:
Fixed in latest commit!